admin_manual/configuration_files/files_access_control.rst

```diff
@@ -49,6 +49,20 @@ This example blocks access to any folder with the tag ``Confidential``.
     .. figure:: images/files_access_control_collaborative_tags.png
        :alt: Deny access based on collaborative tag
 
+Prevent uploading of specific files
+-------------------------
```

Comment:
It doesn't matter as it just creates a separator. I think 4 dashes are the minimum.